spec/dummy/config/initializers/react_on_rails.rb

```diff
@@ -0,0 +1,3 @@
+ReactOnRails.configure do |config|
+  config.bundle_js_file = "app/assets/javascripts/generated/server.js"
```

Comment:
@justin808 good point, I totally agree if we want to cache on server, then there is no any benefit for splitting the file.